### ISSUE TITLE
Compatibility Issues / Only Windows?

### ISSUE THREAD

**Issue by SnivySquid65** [HertzDevil/0CC-FamiTracker#106]
Hello!  I would be happy if you just told me what you used to compile the source, but even then, I guess I should really get to the point.  Also, as I'm righting this, the Famitracker website may have been hacked/sold, because the page is in German now.  I don't speak German.

## I am running Linux
I love Linux!  I understand that Windows is far more popular, but I wonder why there wouldn't be options for Mac, Solaris, or the Unix family of operating systems...?  I did try to run the .exe with the Wine package, (a compatibility layer for Windows apps) but it didn't have sound, and there was an error that prevented me from really using the thing.  Now, this could be an issue with Arch based systems, (running Manjaro, it's Arch based) but I don't think it would be better by running something like Debian or Fedora.

If someone could follow up, that would be great.

Thanks!
-SnivySquid65

**Comment by langel** [comment]
whoever owned the famitracker site let it expire

here is the modern fork most people are contributing to: 
https://github.com/Dn-Programming-Core-Management/Dn-FamiTracker

FamiTracker needs a complete GUI rewrite before it's portable

**Comment by SnivySquid65** [comment]
Yeah, I tried that too, but no download.  I don't know what they used to compile the source, either.  :\

**Comment by langel** [comment]
There are downloads on the release page:
https://github.com/Dn-Programming-Core-Management/Dn-FamiTracker/releases/tag/Dn0.5.0.1

Pretty sure you need MS Visual Studio to compile.
